warehouse/integrations/snowflake/snowflake.go

```diff
@@ -603,63 +643,52 @@ func (sf *Snowflake) copyInto(
 		loadFolder,
 		sf.authString(),
 	)
+	log.Infow("copy command")
 
-	sanitisedQuery, regexErr := misc.ReplaceMultiRegex(sqlStatement, map[string]string{
-		"AWS_KEY_ID='[^']*'":     "AWS_KEY_ID='***'",
-		"AWS_SECRET_KEY='[^']*'": "AWS_SECRET_KEY='***'",
-		"AWS_TOKEN='[^']*'":      "AWS_TOKEN='***'",
-	})
-	if regexErr != nil {
-		sanitisedQuery = ""
+	rows, err := db.QueryContext(ctx, copyStmt)
+	if err != nil {
+		return 0, fmt.Errorf("copy into table: %w", err)
 	}
-	log.Infow("copy command", lf.Query, sanitisedQuery)
+	defer func() {
+		_ = rows.Close()
+	}()
 
-	if _, err := db.ExecContext(ctx, sqlStatement); err != nil {
-		log.Warnw("failure running COPY command",
-			lf.Query, sanitisedQuery,
-			lf.Error, err.Error(),
-		)
-		return fmt.Errorf("copy into table: %w", err)
+	columns, err := rows.Columns()
+	if err != nil {
+		return 0, fmt.Errorf("getting columns: %w", err)
+	}
+	_, index, found := lo.FindIndexOf(columns, func(item string) bool {
+		return item == "rows_loaded"
+	})
+	if !found {
+		return 0, nil
```

Comment:
`if !found` we should at least get a warning, right? We expect it to be there always otherwise there is some kind of regression with the sdk, right? Also, a test here would be nice too.

Reply:
Sure.

Reply:
I am also exploring [COPY_HISTORY](https://docs.snowflake.com/en/sql-reference/functions/copy_history) for this as well.

Reply:
Added a warn log.

Reply:
We can't use [COPY_HISTORY](https://docs.snowflake.com/en/sql-reference/functions/copy_history) since parallel ingestion can also happen at the same time when Rudderstack is loading.

Reply:
Going with the current approach.